docs/sources/panels-visualizations/visualizations/traces/index.md

```diff
@@ -13,16 +13,89 @@ weight: 850
-This panel is currently in beta. Expect changes in future releases.
-{{% /admonition %}}
-
-_Traces_ are a visualization that enables you to track and log a request as it traverses the services in your infrastructure.
+Traces are data that lets you follow a request as it traverses the services in your infrastructure.
+The Traces panel visualizes this data into a diagram that allows you to easily interpret this data.
 
 For more information about traces and how to use them, refer to the following documentation:
 
 - [What are traces](/docs/grafana-cloud/traces)
 - [Tracing in Explore]({{< relref "../../../explore/trace-integration/" >}})
+- [Tempo data source]({{< relref "../../../datasources/tempo/" >}})
 - [Getting started with Tempo](/docs/tempo/latest/getting-started)
 
 {{< figure src="/static/img/docs/explore/trace-view-9-4.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view" >}}
+
+## Add a panel with tracing visualizations
+
+Once you have tracing data available in your Grafana instance, you can add tracing panels to your Grafana dashboards.
+
+Using a dashboard variable, `traceID`, lets you create a query to show specific traces for a given trace ID.
+For more information about dashboard variables, see the [Variables documentation]({{< relref "../../../dashboards/variables" >}}).
+
+### Before you begin
+
+To use this procedure, you need:
+
+- A Grafana instance
+- A Tempo data source connected to your Grafana instance (see [documentation](/docs/grafana/datasources/tempo/))
+
+### Add the Tracing panel query
+
+You need to add the tracing panel to your dashboard and define a query using the Panel editor.
+The query determines the data that is displayed in the panel.
+For more information on the Panel editor, see the [Panel editor]({{< relref "../../panel-editor-overview" >}}) documentation.
+
+This procedure uses dashboard variables and templates to allow us to enter trace IDs which can then be visualized. We’re going to use a variable called `traceId` and add it as a template query.
+
+1. From your Grafana instance, create a new dashboard or go to an existing dashboard where you would like to add tracing panels.
+1. Select **Add visualization** from a new dashboard or select **Add Panel** on an existing dashboard.
+1. Search for and select the appropriate tracing data source.
+1. In the panel search pane on the right, select the **Visualizations** tab, search for and select for Traces.
+1. On the **Panel editor**, enter a **Title** for your trace panel. For more information on the panel editor, see the [Panel editor documentation]({{< relref "../../panel-editor-overview" >}}).
+1. Select the **TraceQL** query type tab from the query editor.
+1. Enter `${traceId}` in the TraceQL query field to create a dashboard variable. This variable is used as the template query. <br /> {{< figure src="/static/img/docs/panels/traces/screenshot-traces-template-query.png" caption="Add a template query" >}}
```

Comment:
This looks like there is another way to fix this besides inserting HTML. What  is the indentation were two spaces?

Reply:
I have been trying to figure out another way, and so far using br has been the only thing that worked. I tried using two indents at first. Using indentation with two spaces caused the images to have a number.

Reply:
I wonder it this has something to do with using a shortcode instead of the markdown image syntax.

Reply:
Got this sorted with help from Jack!